src/diagnostics/gravitywave_diagnostics.jl

```diff
@@ -0,0 +1,86 @@
+# This file is included in diagnostics.jl
+
+# Gravity-wave related diagnostics
+
+###
+# Eastward Non-Orographic Gravity-Wave Tendency 
+###
+compute_utendnogw!(out, state, cache, time) = compute_utendnogw!(
+    out,
+    state,
+    cache,
+    time,
+    cache.atmos.non_orographic_gravity_wave,
+)
+compute_utendnogw!(_, _, _, _, non_orographic_gravity_wave::T) where {T} =
+    error_diagnostic_variable("utendnogw", non_orographic_gravity_wave)
+
+function compute_utendnogw!(
+    out,
+    state,
+    cache,
+    time,
+    ::T,
+) where {T <: NonOrographicGravityWave}
+    if isnothing(out)
+        # u_waveforcing is not a field:
+        # return Fields.array2field(
+        #     cache.non_orographic_gravity_wave.u_waveforcing,
+        #     axes(state.c),
+        # )
+        return copy(cache.non_orographic_gravity_wave.u_waveforcing)
```

Comment:
```suggestion
        return copy(cache.non_orographic_gravity_wave.uforcing)
```
(same for other diagnostics)